addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1427,7 +1431,9 @@ const Wysiwyg = Widget.extend({
                 if (!this.showTooltip || $target.attr('title') !== undefined) {
                     return;
                 }
+                this.odooEditor.observerUnactive();
                 $target.tooltip({title: _t('Double-click to edit'), trigger: 'manual', container: 'body'}).tooltip('show');
+                this.odooEditor.observerActive();
                 setTimeout(() => $target.tooltip('dispose'), 800);
```

Comment:
Didn't check, just reading code on another PR trying to not listen the tooltip.
But isn't the tooltip dispose going to be listened by the observer / flag stuff as dirty as it will be done after the observer was re-activated?